components/Layout/Footer/types.ts

```diff
@@ -0,0 +1,5 @@
+type FooterProps = {
+    lastPublishedAt: string;
+};
+
+export default FooterProps;
```

Comment:
Storing types separately is a good way of keeping file size down, do you feel it interferes with your development much? The projects I've worked on so far mostly keep the prop types beside the component, with the exception of the extended page types I've used in the careers site

Reply:
> do you feel it interferes with your development much?

Not especially; some components come with a lot of typing so I preferred having the same approach (of having a separate file for types) in all components.